src/lib/build.ts

```diff
@@ -1,70 +1,89 @@
 import fs from 'fs'
 import process from 'process'
 
-import build from '@netlify/build'
+import build, { type OnEnd, type OnPostBuild } from '@netlify/build'
 // @ts-expect-error TS(7016) FIXME: Could not find a declaration file for module 'toml... Remove this comment to see the full error message
 import tomlify from 'tomlify-j0.4'
+import type { OptionValues } from 'commander'
 
 import { getFeatureFlagsFromSiteInfo } from '../utils/feature-flags.js'
 
 import { getBootstrapURL } from './edge-functions/bootstrap.js'
 import { featureFlags as edgeFunctionsFeatureFlags } from './edge-functions/consts.js'
 
-/**
- * The buildConfig + a missing cachedConfig
- * @typedef BuildConfig
- * @type {Parameters<import('@netlify/build/src/core/main.js')>[0] & {cachedConfig: any}}
```

Comment:
this wasn't doing anything, because `src/` should be `lib/` and it exports `buildSite`, not a default export... 🤦🏼